src/components/core/Input/Input.module.scss

```diff
@@ -62,5 +77,9 @@
 
     top: 0;
     transition: 0.2s;
+
+    &--error {
+      color: $color-red;
+    }
```

Comment:
This file is not nested per requirement I set and I told you not to use `~` selector